test/apidemos/appium/appiumTest.js

```diff
@@ -17,7 +17,8 @@ describe('appium api demo', function () {
     'isKeyboardShown',
     'getContexts',
     'getContext',
-    'setContext'
+    'setContext',
+    're'
```

Comment:
```suggestion
    'resetApp'
```

But I wonder why the test is not failing because of this?